run.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 source ./venv/bin/activate
 # Wikidp production token
-export CONSUMER_TOKEN=''
-export SECRET_TOKEN=''
+# export WIKIDP_CONSUMER_KEY=''
+# export WIKIDP_CONSUMER_SECRET=''
```

Comment:
based on our conversation, commenting these out to prevent overwriting exisiting env vars.